content/en/developers/ide_integrations/vscode/_index.md

```diff
@@ -10,25 +10,33 @@ further_reading:
 - link: "/continuous_testing/testing_tunnel/"
   tag: "Documentation"
   text: "Learn about the Continuous Testing Tunnel"
+- link: "/integrations/guide/source-code-integration/"
+  tag: "Documentation"
+  text: "Learn about Source Code Integration."
 ---
 
 {{< callout url="#" btn_hidden="true">}}
-  The Datadog extension for Visual Studio Code is in public beta. It is intended for Datadog users who have <a href="https://docs.datadoghq.com/synthetics/#pagetitle">Synthetic tests</a> set up already. The extension supports running <a href="https://docs.datadoghq.com/synthetics/api_tests/http_tests">HTTP tests</a> and <a href="https://docs.datadoghq.com/synthetics/browser_tests">browser tests</a>. If the extensions stops working unexpectedly, check for extension updates or <a href=#feedback>reach out to the team</a>.
+  The Datadog extension for VS Code is in public beta. It is intended for all developers that use Datadog products, and especially those using <a href="https://docs.datadoghq.com/synthetics/#pagetitle">Synthetic tests</a>. If the extension stops working unexpectedly, check for extension updates or <a href=#feedback>reach out to the team</a>.
 {{< /callout >}}
 
 ## Overview
```

Comment:
We have a section for `View in VS Code` but there's no mention of it in the overview section.

Reply:
Added